playbook/app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.scss

```diff
@@ -3,6 +3,320 @@
 @import "../tokens/opacity";
 @import "../pb_avatar/avatar";
 
+$avatar-sizes: (
```

Comment:
Writing out the pixel sizes is good because it clearly communicates what size things need to be. 

We don't need to add a token for 9px and its more confusing to have raw pixel values next to tokens.